packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -74,22 +125,43 @@ const RelationInput = ({
               onMenuOpen={onSearchOpen}
               onMenuScrollToBottom={onSearchNextPage}
               placeholder={placeholder}
+              name={name}
             />
           </>
         }
         loadMore={
           !disabled &&
           labelLoadMore && (
-            <TextButton onClick={() => onRelationLoadMore()} startIcon={<Refresh />}>
+            <TextButton
+              disabled={relations.isLoading}
+              onClick={() => onRelationLoadMore()}
+              startIcon={
+                relations.isLoading ? (
+                  // To replace with loading prop on TextButton after DS release
+                  <LoaderWrapper>
+                    <Loader />
+                  </LoaderWrapper>
+                ) : (
+                  <Refresh />
+                )
+              }
+            >
               {labelLoadMore}
             </TextButton>
           )
         }
       >
-        <RelationList height={listHeight}>
-          {relations.isSuccess &&
-            relations.data.pages.flat().map((relation) => {
-              const { publicationState, href, mainField, id } = relation;
+        <RelationList overflow={totalNumberOfRelations > numberOfRelationsToDisplay && overflow}>
```

Comment:
This can lead to a boolean value (false), which is not reflected in the prop-types. Could we make the default an empty string above, so that you don't have to add any condition here?

Reply:
will do for now, I think I used it to make sure we never display shadows if we have not enough items to display the scrolling list UI - but it might change when we'll fix the shadows anyway!